tests/constraints.txt

```diff
@@ -5,6 +5,7 @@ tqdm==4.56.0
 jsonschema==3.0.0
 jsonpatch==1.15
 pyyaml==5.1
+importlib_resources==1.3.0
```

Comment:
I've verified that this passes the Minimum supported dependencies workflow, so LGTM. :+1: